<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DiaryStats</class>
 <widget class="QWidget" name="DiaryStats">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>650</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1250</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout_4">
     <property name="leftMargin">
      <number>9</number>
     </property>
     <property name="topMargin">
      <number>9</number>
     </property>
     <property name="rightMargin">
      <number>9</number>
     </property>
     <property name="bottomMargin">
      <number>9</number>
     </property>
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Charts</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <layout class="QGridLayout" name="gridLayout_11">
        <item row="0" column="0">
         <widget class="QTabWidget" name="charts">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <property name="movable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="spline">
           <attribute name="title">
            <string>Line</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_2">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <property name="spacing">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QChartView" name="spline_chart_view">
              <property name="styleSheet">
               <string notr="true">border: none;</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="pie">
           <attribute name="title">
            <string>Pie</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_7">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <property name="spacing">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QChartView" name="pie_chart_view">
              <property name="styleSheet">
               <string notr="true">border: none;</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="polar">
           <attribute name="title">
            <string>Polar</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_3">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <property name="spacing">
             <number>0</number>
            </property>
            <item row="1" column="0">
             <widget class="QChartView" name="polar_chart_view">
              <property name="styleSheet">
               <string notr="true">border: none;</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Comparison</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <layout class="QGridLayout" name="gridLayout_10">
        <item row="0" column="2">
         <widget class="QGroupBox" name="groupBox_5">
          <property name="title">
           <string>Difference</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <layout class="QGridLayout" name="gridLayout_15">
           <property name="leftMargin">
            <number>15</number>
           </property>
           <property name="rightMargin">
            <number>15</number>
           </property>
           <property name="spacing">
            <number>15</number>
           </property>
           <item row="0" column="1">
            <widget class="QLabel" name="d1">
             <property name="minimumSize">
              <size>
               <width>40</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>2</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="DiaryComparisonLabel" name="ok_3">
             <property name="text">
              <string>5</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="DiaryComparisonLabel" name="very_bad_3">
             <property name="text">
              <string>1</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="DiaryComparisonLabel" name="very_good_3">
             <property name="text">
              <string>9</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="d4">
             <property name="text">
              <string>8</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="DiaryComparisonLabel" name="bad_3">
             <property name="text">
              <string>3</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="d3">
             <property name="text">
              <string>6</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="DiaryComparisonLabel" name="good_3">
             <property name="text">
              <string>7</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLabel" name="d5">
             <property name="text">
              <string>10</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLabel" name="d0">
             <property name="text">
              <string>12</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="DiaryComparisonLabel" name="unknown_3">
             <property name="text">
              <string>11</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="d2">
             <property name="text">
              <string>4</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="DiaryComparisonLabel" name="starred_3">
             <property name="text">
              <string>13</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QLabel" name="ds">
             <property name="text">
              <string>14</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>This month</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <layout class="QGridLayout" name="gridLayout_13">
           <property name="leftMargin">
            <number>15</number>
           </property>
           <property name="rightMargin">
            <number>15</number>
           </property>
           <property name="spacing">
            <number>15</number>
           </property>
           <item row="2" column="1">
            <widget class="QLabel" name="t3">
             <property name="text">
              <string>6</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLabel" name="t5">
             <property name="text">
              <string>10</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLabel" name="t0">
             <property name="text">
              <string>12</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="DiaryComparisonLabel" name="unknown">
             <property name="text">
              <string>11</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QLabel" name="ts">
             <property name="text">
              <string>14</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="DiaryComparisonLabel" name="very_bad">
             <property name="text">
              <string>1</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="DiaryComparisonLabel" name="very_good">
             <property name="text">
              <string>9</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="DiaryComparisonLabel" name="good">
             <property name="text">
              <string>7</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="DiaryComparisonLabel" name="ok">
             <property name="text">
              <string>5</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="t4">
             <property name="text">
              <string>8</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="t1">
             <property name="minimumSize">
              <size>
               <width>40</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>2</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="DiaryComparisonLabel" name="bad">
             <property name="text">
              <string>3</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="t2">
             <property name="text">
              <string>4</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="DiaryComparisonLabel" name="starred">
             <property name="text">
              <string>13</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="1">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>Last month</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <layout class="QGridLayout" name="gridLayout_14">
           <property name="leftMargin">
            <number>15</number>
           </property>
           <property name="rightMargin">
            <number>15</number>
           </property>
           <property name="spacing">
            <number>15</number>
           </property>
           <item row="0" column="0">
            <widget class="DiaryComparisonLabel" name="very_bad_2">
             <property name="text">
              <string>1</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="l1">
             <property name="minimumSize">
              <size>
               <width>40</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>2</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="DiaryComparisonLabel" name="very_good_2">
             <property name="text">
              <string>9</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLabel" name="l5">
             <property name="text">
              <string>10</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="DiaryComparisonLabel" name="unknown_2">
             <property name="text">
              <string>11</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLabel" name="l0">
             <property name="text">
              <string>12</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="l2">
             <property name="text">
              <string>4</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="l3">
             <property name="text">
              <string>6</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="DiaryComparisonLabel" name="bad_2">
             <property name="text">
              <string>3</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="DiaryComparisonLabel" name="good_2">
             <property name="text">
              <string>7</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="DiaryComparisonLabel" name="ok_2">
             <property name="text">
              <string>5</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="l4">
             <property name="text">
              <string>8</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="DiaryComparisonLabel" name="starred_2">
             <property name="text">
              <string>13</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QLabel" name="ls">
             <property name="text">
              <string>14</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_6">
     <property name="horizontalSpacing">
      <number>10</number>
     </property>
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <item row="0" column="1">
      <widget class="QPushButton" name="prev_month">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Previous month.</string>
       </property>
       <property name="text">
        <string>&lt;</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="4">
      <widget class="QDateEdit" name="year_edit">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Year edit</string>
       </property>
       <property name="displayFormat">
        <string>yyyy</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="7">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="3">
      <widget class="QComboBox" name="month_dropdown">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Month dropdown</string>
       </property>
       <item>
        <property name="text">
         <string>January</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Feburary</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>March</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>April</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>May</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>June</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>July</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>August</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>September</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>October</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>November</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>December</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QPushButton" name="next_month">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Next month.</string>
       </property>
       <property name="text">
        <string>&gt;</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QtCharts</header>
  </customwidget>
  <customwidget>
   <class>DiaryComparisonLabel</class>
   <extends>QLabel</extends>
   <header location="global">src/util/diarycomparisonlabel.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>prev_month</tabstop>
  <tabstop>month_dropdown</tabstop>
  <tabstop>year_edit</tabstop>
  <tabstop>next_month</tabstop>
  <tabstop>spline_chart_view</tabstop>
  <tabstop>pie_chart_view</tabstop>
  <tabstop>polar_chart_view</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
